Q/QhullMiniWrapper/build_tarballs.jl

```diff
@@ -16,7 +16,7 @@ cd $WORKSPACE/srcdir/
 ${CC} -fPIC -shared -o ${libdir}/qhullwrapper.${dlext} qhullwrapper.c -L${libdir} -lqhull_r
 """
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = supported_platforms(; experimental=true)
```

Comment:
Ah, I thought it would potentially include lower-tier support platforms (specifically ARM Macs - but I guess now they are also Tier 1 as of this morning). If it's a no-op, would it be OK to keep them or should it be removed before the merge?

Reply:
Let me double check - I am not using the package directly myself, but the package I _am_ trying to use reports only seeing the x86_64 version of MiniQhullWrapper (and Qhull). Let me double check.

Reply:
Perhaps this is not a place to discuss this, but here's what I'm trying to do. I'm using `Ripserer.jl` and trying to precompile it (julia v.1.9 on ARM mac) which loads `MiniQhull` (0.4.0 with the most recent UUID). Precompile of Ripserer fails when `using MiniQhull` is invoked. When I manually add MiniQhull and issue `using MiniQhull` in my REPL, I get the same error as in precompile of `Ripserer.jl`: only `x86_64` architecture version of `libMiniQhullWrapper` is found, but `arm64` is requested (`mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64')`). I assumed this was due to `arm64` being globally unavailable, but you're right - it looks like the platform is available on Yggdrasil.

Would this then be more appropriate discussion for a `MiniQhull`'s Github ticket?